src/main/java/com/thirdcc/webapp/web/rest/UserResource.java

```diff
@@ -226,8 +226,21 @@ public UserResource(UserService userService, UserRepository userRepository, Mail
      * @return the {@link ResponseEntity} with status {@code 200 (OK)} and the list of users in body.
      */
     @GetMapping("/users/event-crews/{eventId}")
-    public List<UserDTO> getNotEventCrewUsers (Pageable pageable, @PathVariable Long eventId) {
+    public ResponseEntity<List<UserDTO>> getNotEventCrewUsers (Pageable pageable, @PathVariable Long eventId) {
         log.debug("REST request to get not event crew user of event: {}", eventId);
-        return userService.getNotEventCrewUsers(pageable, eventId);
+        List<UserDTO> users = userService.getNotEventCrewUsers(pageable, eventId);
+        return ResponseEntity.ok().body(users);
+    }
+
+    /**
+     * {@code GET /users/family} : get all users that have no assigned family.
+     * @param hasFamily checking user's UserCCInfo is exist or not
+     * @return the {@link ResponseEntity} with status {@code 200 (OK)} and the list of users in body.
+     */
+    @GetMapping(value="/users", params="family")
+    public ResponseEntity<List<UserDTO>> getUsersWithFamilyCheck (Pageable pageable, @RequestParam("family") Boolean hasFamily){
```

Comment:
this API URL suppose to look like which one?
- /users/family
- /users?family=true
- /users/family?family=true

Will this API URL be conflict with getAllUsers API?

Reply:
updated the API URL in 97be5e67376af81fe0da033df6c9bd60837a73e6, the new URL will be:
- /users/family?hasFamily=true